addons/web_editor/static/src/js/wysiwyg/widgets/select_options_popover_widget.js

```diff
@@ -0,0 +1,78 @@
+/** @odoo-module **/
+
+import Widget from 'web.Widget';
+
+const SelectOptionsPopoverWidget = Widget.extend({
+    template: 'wysiwyg.widgets.selectoptions.edit.tooltip',
+    xmlDependencies: ['/web_editor/static/src/xml/wysiwyg.xml'],
+    events: {
+        'click .o_we_remove_select': '_onRemoveSelectClick',
+        'click .o_we_edit_select': '_onEditSelectClick',
+    },
+
+    /**
+     * @constructor
+     * @param {Element} target: target Element for which we display a popover
+     * @param {OdooClass} wysiwyg
+     */
+    init(parent, target, wysiwyg) {
+        this._super(...arguments);
+        this.target = target;
+        this.$target = $(target);
+        this.wysiwyg = wysiwyg
+    },
+    /**
+     * @override
+     */
+    start() {
+        this.$target.popover({
+            html: true,
+            delay: 2000,
```

Comment:
why 2000? is this standard ? should this be optionable ?

Reply:
Good question. I'll look elsewhere in the codebase and do the same.